src/example-relay/src/graphql/types/interface/Error.js

```diff
@@ -0,0 +1,11 @@
+// @flow strict
```

Comment:
Is the `interface` folder needed? Maybe put it one level up until it's really needed.